sns/parity_math.c

```diff
@@ -1426,14 +1442,51 @@ M0_INTERNAL void m0_parity_math_buffer_xor(struct m0_buf *dest,
 			(m0_parity_elem_t)((uint8_t*)src[0].b_addr)[ei];
 }
 
+#if ISAL_ENCODE_ENABLED
+static int isal_ir_init(struct m0_sns_ir *ir)
+{
+	int ret;
+
+	M0_PRE(ir != NULL);
+
```

Comment:
Do we need to do,
```
M0_SET0(ir);
```
To make sure all members have default value assigned, we need to clear the memory region.